help/en/html/doc/Technical/JVMCapabilities.shtml

```diff
@@ -340,10 +340,14 @@
       finds numerous pages that show how to install Java 1.7 on XP
       or XP SP2.</p>
 
-      <p>Java 9 is not released yet, but pre-release information is
-      available on the <a href=
-      "https://jdk9.java.net/jdk9_supported_platforms.html">supported
-      platforms page</a>.</p>
+      <p>Oracle has announced that "Java SE 9 will be a short term release, and users should immediately transition to the next release (18.3) when available".
+      That 18.3 refers to the statement "Oracle has proposed a new version scheme (YY.M) starting in March, 2018", 
+      hence 18.3 for that version. But they've also said that: "
+      Java SE 18.3 will be a short term release and users should transition to the next release when available"
+      So there's some uncertainty coming in the Java releases. For more info, please
```

Comment:
I don't think there is uncertainty, because they also state that long term support releases are every three years, and that 18.9 will be the first long term support release (supported from September 2018 to September 2023). This is on the page you quote from.

Reply:
I don't think it's at all certain whether JMRI will explicitly support 18.3 or wait to 18.9.

Reply:
I think we have to ensure JMRI at least runs on the current LTS and feature release versions, but only consider requiring features from an LTS version. Its not unlikely that, similar to the need to support Java 9 for Windows users with HiDPI displays, we will find ourselves supporting the current shipping feature release for hardware or OS support reasons.

Reply:
Agreed.